community/connections/AnyRail/index.shtml

```diff
@@ -139,9 +139,11 @@
       script may be useful when doing large migrations with block sensors.
       See the associated
       <a href="./AnyRailBuildBlockSensorList.pdf">PDF instructions</a>
+      (Original ODT file
+      <a href="/AnyRailBuildBlockSensorList.odt">.odt file</a>)
```

Comment:
Sorry for being nitpicky, but I suggest that you add:
... that can be edited by Microsoft Word, LibreOffice or OpenOffice.

People might not know what .odt is. (I didn't know myself, but I did a Google search before I gave the suggestion).

But thanks for adding the .odt file.